README.md

```diff
@@ -112,6 +112,27 @@ install
 
 Please refer to the CMake Manual for more details.
 
+**Windows**:
+Different configurations have different run-time requirements.  The `Debug`
+configuration uses the windows debug C runtime, whereas the `RelWithDebInfo`
+does not.  This may determine which version you can use with your executable.
+
+Some special notes regarding `cmake`:
+- cmake doesn't need the configuration int the `-B` step since it creates
+  all the `*.vcxproj` files at once.
```

Comment:
What do you mean by that? The `-B` flag specifies the output directory of the VC project, otherwise it would just clutter the working directory.

Reply:
Sorry, that was unclear.  You don't need the `-DCMAKE_BUILD_TYPE=RelWithDebInfo` setting there, although it is harmless.  All the configurations are embedded in the resulting .vcxproj files.

Reply:
You do not need it in case of MSVS generator, but you can use `-G Ninja` for example

Reply:
Right you are.  So it may well be left in place though.  It is harmless enough :)